pkgs/development/idris-modules/TODO.md

```diff
@@ -1,3 +1,3 @@
 * Build the RTS separately from Idris
 * idris2nix
-* Only require gmp, rts when compiling executables
```

Comment:
GitHub does not show the final missing new line here. Maybe an error?

Reply:
Seems to be a GitHub issue, the newline change shows up in Git.